src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -470,20 +470,20 @@ private void ParseNLogElement(ILoggingConfigurationElement nlogElement, string f
         /// <summary>
         /// Parses a single config section within the NLog-config
         /// </summary>
-        /// <param name="section"></param>
+        /// <param name="configSection"></param>
         /// <returns>Section was recognized</returns>
-        protected override bool ParseNLogSection(ILoggingConfigurationElement section)
+        protected override bool ParseNLogSection(ILoggingConfigurationElement configSection)
```

Comment:
unfortunately this is a binary breaking change

Reply:
Pretty sure we have not released NLog 4.6 yet

Reply:
ah yes, then it's OK indeed :)